client/package.json

```diff
@@ -36,6 +36,7 @@
     "ember-cli-eslint": "3.0.0",
     "ember-cli-flash": "1.3.17",
     "ember-cli-focus-input": "1.0.0",
+    "ember-cli-google-analytics": "^1.5.0",
```

Comment:
Correct for non-registered users! Once we have the user intercom is the way to go.